src/routers/staff.router.ts

```diff
@@ -1,3 +1,152 @@
-import { Router } from "express";
+import { Router, json } from "express";
+
+const {
+  createStaff,
+  updateStaff,
+  updateStaffActive,
+  getAllStaff,
+  getStaffByEmail,
+  getStaffByID,
+  getStaffByActive,
+  getStaffByProgram,
+} = require("../controllers/staff.controller");
 
 export const staffRouter = Router();
+staffRouter.use(json());
+
+// POST add new Staff document
+staffRouter.post("/", async (req, res) => {
+  try {
+    let newStaff = await createStaff(req.body);
+    res.status(201).send(newStaff);
+  } catch (err) {
+    res.status(500).send(err);
+  }
+});
+
+// GET all Staff documents
+staffRouter.get("/", async (_req, res) => {
+  try {
+    let allStaff = await getAllStaff();
+    res.status(200).send(allStaff);
+  } catch (err) {
+    res.status(404).send(err);
```

Comment:
Raw errors probably shouldn't be sent directly back to the user: https://stackoverflow.com/questions/67290510/whether-all-error-messages-should-come-from-the-backend-in-a-form-that-can-be-sh 

Maybe this should be logged then a cleaner human readable version should be sent to the user. @samshefnie do we have a clean way of logging things to a file set up for this project yet?